.gitignore

```diff
@@ -1,3 +1,60 @@
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided this notice is
+# preserved.  This file is offered as-is, without any warranty.
+# Names of contributors must not be used to endorse or promote products
```

Comment:
This seems to be not english :) and I would prefer avoiding this at all, we already have a global license